crt_portal/cts_forms/tests.py

```diff
@@ -443,6 +451,33 @@ def test_text_filter(self):
         self.assertEqual(len(disjointed_phrase_response), self.len_all_results)
         self.assertEqual(len(url_not_in_phrase_response), 0)
 
+    def test_first_name_filter(self):
```

Comment:
This looks good. I think we can write fewer tests for each filter going forward since lots of the underlying mechanics are shared with other tested examples.